Source/Module/Account/AccountScreen.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AccountScreen.swift
+//  GithubNotifications
+//
+//  Created by Mark G on 21/09/2021.
+//  Copyright © 2021 Nimblehq. All rights reserved.
+//
+
+import SwiftUI
+
+struct AccountScreen: View {
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            // TODO: Update with real data in integrate
+            Text("Username")
+                .foregroundColor(.brightGray)
+            HStack {
+                Text("@suho")
+                    .font(.body.bold())
+                Spacer()
+                Button {} label: {
+                    Text("Sign Out" )
```

Comment:
there's an extra space here
```suggestion
                    Text("Sign Out")
```